agents/langchain-deepagents-code/managed-dcode-runtime.py

```
@@ -1309,6 +1338,495 @@ def managed_auto_approval_enabled() -> bool:
+                target = output[key.data]
+                remaining = maximum_output + 1 - sum(len(value) for value in output.values())
+                if remaining > 0:
+                    target.extend(chunk[:remaining])
+                if sum(len(value) for value in output.values()) > maximum_output:
+                    terminal_status = "output_limit_exceeded"
+                    _terminate_validation_process(process)
+            if terminal_status is not None and process.poll() is not None:
+                for key in list(selector.get_map().values()):
+                    selector.unregister(key.fileobj)
+        return_code = process.wait(timeout=1)
+    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)